lerna.json

```diff
@@ -4,9 +4,6 @@
   ],
   "useNx": false,
   "version": "0.0.0",
-  "packages": [
-    "packages/*"
-  ],
```

Comment:
This ended up as a cleanup, from what I understand of [lerna's quickstart](https://lerna.js.org/docs/getting-started) we need to define the `workspaces` for it to work.

In multiple guides like this one, https://www.honeybadger.io/blog/monorepo-yarn-workspace-lerna/ and other projects I've seen the `workspaces` duplicate. I can try to remove it, but it will probably break the monorepo.

Reply:
yeah it's just that the docs say that [Lerna relies on the workspaces setting defined for the package manager](https://lerna.js.org/docs/api-reference/configuration#packages) (i.e., npm, yarn)

Reply:
Plus the Honeybadger blog, if I'm reading it correctly, instructs the reader to put workspaces in package.json not lerna.json